src/app/mod.rs

```diff
@@ -284,7 +293,7 @@ impl<S: Default + ProvableStore + 'static> Application for BaseCoinApp<S> {
     fn commit(&self) -> ResponseCommit {
         let mut modules = self.modules.write().unwrap();
         for m in modules.iter_mut() {
-            m.commit().expect("failed to commit to state");
+            m.store().commit().expect("failed to commit to state");
```

Comment:
Thanks for reviewing, @thanethomson! 🙏 
I replaced the `commit()` method with the `store()` method because some modules need to share their module stores with gRPC service instances. I am working on a better design for this on the `typed-stores` branch - I'll document this in a separate issue.